app/main/controllers/review-submission-ctrl.js

```diff
@@ -1,8 +1,177 @@
 'use strict';
-angular.module('myApp')
-.controller('ReviewSubmissionCtrl', function ($scope, $state) {
+ angular.module('myApp')
+.controller('ReviewSubmissionCtrl', function ($scope, $http, $state, DataService, $document,$uibModal, $log, $rootScope, $filter, SignaturePad) {
 
+var $ctrl = this;
 
+  $scope.submit = function(isValid) {
 
+      $scope.submitted = true;
 
+      if( $rootScope.application.request_content.review.signature !== undefined) {
+
+            console.log("cash money");
+        //  $state.go('home');
+      }
+        else if (isValid == false) {
+          console.log("Valid Check failed");
+
+      }
+          else {
+        $ctrl.open();
+      }
+
+    };
+
+
+  $ctrl.animationsEnabled = true;
+
+  $ctrl.open = function (size, parentSelector) {
+    var parentElem = parentSelector ?
+      angular.element($document[0].querySelector('.modal-demo ' + parentSelector)) : undefined;
```

Comment:
.modal-demo doesn't seem to exist anywhere -- does that mean that parentElem is always undefined? If so let's clean that up.